hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerStateMachine.java

```diff
@@ -718,4 +718,7 @@ public long getTermForIndex(long transactionIndex) {
     return applyTransactionMap.get(transactionIndex);
   }
 
+  public void awaitDoubleBufferFlush() throws InterruptedException {
```

Comment:
No, it will be used by the tarball creation code.  That code will first block the deletion service, (which is my next pr.)  Then call this method to wait until the buffers are cleared.  That way we can be sure that no moveKey operations from the deletion service are waiting in the buffers